drivers/overlay/overlay.go

```diff
@@ -620,7 +620,7 @@ func supportsOverlay(home string, homeMagic graphdriver.FsMagic, rootUID, rootGI
 		if len(flags) < unix.Getpagesize() {
 			err := unix.Mount("overlay", mergedDir, "overlay", 0, flags)
 			if err == nil {
-				logrus.Errorf("overlay test mount with multiple lowers failed, but succeeded with a single lower")
+				logrus.StandardLogger().Logf(logLevel, "overlay test mount with multiple lowers failed, but succeeded with a single lower")
```

Comment:
I am a bit confused. Why not use `.Debugf`?

Reply:
logLevel is set to `logrus.ErrorLevel` for root and `logrus.DebugLevel` for rootless, so it is still an error when running as root